requirements.txt

```diff
@@ -23,3 +23,13 @@ scikit-optimize==0.5.2
 
 # Required for plotting data
 #plotly==3.0.0
+
+# Required for plotting data
+plotly==3.0.0
+
+# find first, C search in arrays
+py_find_1st==1.1.1
+
+#Load ticker files 30% faster
+ujson==1.35
```

Comment:
not sure about this files changes

ujson has it's own PR - and is not used by develop atm, so i would not force it here.
Also, py_find_1st should be included in the PR requiring it (which i suspect will be backslash?)

plotly on the other hand is still there as optional dependency - not needed in a docker-container at all...